<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1018</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0" rowspan="2" colspan="2">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>hinh-anh-cac-mon-an-ngon-nhan-lam-theo-yeu-cau-rieng.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="1" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>205</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="QWidget" name="widget" native="true">
         <property name="maximumSize">
          <size>
           <width>430</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">;

background-color: rgb(255, 255, 255);
border-radius:20px</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="3" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>10</width>
              <height>10</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="2">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>10</width>
              <height>10</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1">
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>10</width>
              <height>10</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="0">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>10</width>
              <height>10</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1" rowspan="2">
           <widget class="QWidget" name="widget_4" native="true">
            <property name="styleSheet">
             <string notr="true">background-color: qlineargradient(
    spread:pad, 
    x1:0, y1:0, x2:1, y2:0, 
    stop:0 #F2994A,  /* Màu đầu tiên */
    stop:1 #F2C94C   /* Màu cuối cùng */
);
qqq

border-radius:20px</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>6</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>22</number>
             </property>
             <item>
              <widget class="QWidget" name="widget_6" native="true">
               <property name="maximumSize">
                <size>
                 <width>400</width>
                 <height>80</height>
                </size>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item alignment="Qt::AlignHCenter">
                 <widget class="QLabel" name="label_7">
                  <property name="styleSheet">
                   <string notr="true">color: #356a9c;
font: 20pt &quot;Bahnschrift&quot;;
background-color: rgb(255, 255, 255);</string>
                  </property>
                  <property name="text">
                   <string>Login Here</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="maximumSize">
                <size>
                 <width>400</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 12pt &quot;Bahnschrift&quot;;
color: rgb(255, 255, 255);
</string>
               </property>
               <property name="text">
                <string>Full name</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="usernameLineEdit_2">
               <property name="styleSheet">
                <string notr="true">border: 1px solid #ccc; border-radius: 5px; padding: 5px; background-color: #f9f9f9;</string>
               </property>
               <property name="placeholderText">
                <string>Tên người dùng</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 12pt &quot;Bahnschrift&quot;;
color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>Email</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="emailLineEdit_3">
               <property name="styleSheet">
                <string notr="true">border: 1px solid #ccc; border-radius: 5px; padding: 5px; background-color: #f9f9f9;</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
               <property name="placeholderText">
                <string>Email.com</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 12pt &quot;Bahnschrift&quot;;
color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>Password</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="passwordLineEdit_2">
               <property name="styleSheet">
                <string notr="true">border: 1px solid #ccc; border-radius: 5px; padding: 5px; background-color: #f9f9f9;</string>
               </property>
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
               <property name="placeholderText">
                <string>Mật khẩu</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox">
               <property name="maximumSize">
                <size>
                 <width>450</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 9pt &quot;Bahnschrift&quot;;
color: #575556;</string>
               </property>
               <property name="text">
                <string>I have read and agree to MyApp's privacy Policy</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btlogin_2">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>50</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>50</height>
                </size>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">
QPushButton{border-radius: 15px;
font: 600 16pt &quot;Bahnschrift&quot;;
background-color: rgb(88, 184, 69);
color: rgb(255, 255, 255);
}

QPushButton:hover{
	color: rgb(85, 255, 255);
	background-color: rgb(246, 225, 255);
}
QPushBotton{
background-color: rgb(255, 255, 0);
color:white;}</string>
               </property>
               <property name="text">
                <string>Log In</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="font">
                  <font>
                   <family>Bahnschrift</family>
                   <pointsize>12</pointsize>
                   <weight>50</weight>
                   <italic>false</italic>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">font: 12pt &quot;Bahnschrift&quot;;
color: rgb(255, 255, 255);</string>
                 </property>
                 <property name="text">
                  <string>No account</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btregister33">
                 <property name="minimumSize">
                  <size>
                   <width>52</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>150</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">
QPushButton{font: 12pt &quot;Bahnschrift&quot;;
color: #356a9c;
background-color: rgba(255, 0, 0, 0);background-color: white;
border-radius: 15px}

QPushButton:hover{
	color: rgb(85, 255, 255);
	background-color: rgb(246, 225, 255);
}
QPushBotton{
background-color: rgb(255, 255, 0);
color:white;}
</string>
                 </property>
                 <property name="text">
                  <string>Register Here</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>205</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>93</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>93</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1018</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
